--- a/3rdparty/tbb/CMakeLists.txt
+++ b/3rdparty/tbb/CMakeLists.txt
@@ -8,7 +8,7 @@ endif()
 set(tbb_ver "tbb44_20160128oss")
 set(tbb_filename "4.4.3.tar.gz")
 set(tbb_subdir "tbb-4.4.3")
-set(tbb_md5 "8e7200af3ac16e91a0d1535c606a485c")
+set(tbb_md5 "8e3e39e1fdfb3f7c3a5ac8ec1afe186e")
 set(tbb_version_file "version_string.ver")
 ocv_warnings_disable(CMAKE_CXX_FLAGS /wd4702)
 ocv_warnings_disable(CMAKE_CXX_FLAGS -Wshadow)
